$MirOS: ports/www/dillo/stable/patches/patch-dpid_main_c,v 1.2 2005/06/19 17:20:19 bsiegert Exp $
$OpenBSD: patch-dpid_main_c,v 1.1 2004/02/21 01:37:19 couderc Exp $
--- dpid/main.c.orig	Mon Jan  2 18:33:55 2006
+++ dpid/main.c	Mon May 15 17:49:33 2006
@@ -69,7 +69,7 @@ static int start_filter_plugin(struct dp
    }
    if ( pid == 0) {
       /* Child, start plugin */
-      if (execl(dpi_attr.path, dpi_attr.path, NULL) == -1) {
+      if (execl(dpi_attr.path, dpi_attr.path, (void *)NULL) == -1) {
          ERRMSG("start_plugin", "execl", errno);
          fprintf(stderr, "ERROR in child proc for %s\n", dpi_attr.path);
          exit(1);
@@ -101,7 +101,7 @@ static void start_server_plugin(struct d
       fprintf(stderr, "ERROR in child proc for %s\n", dpi_attr.path);
       exit(1);
    }
-   if (execl(dpi_attr.path, dpi_attr.path, NULL) == -1) {
+   if (execl(dpi_attr.path, dpi_attr.path, (void *)NULL) == -1) {
       ERRMSG("start_plugin", "execl", errno);
       fprintf(stderr, "ERROR in child proc for %s\n", dpi_attr.path);
       exit(1);
